grails-app/views/organization/edit.gsp

```diff
@@ -18,16 +18,16 @@
 	            </div>
             </g:hasErrors>
 
-			<div class="button-bar">
-				<g:link class="button" action="list">
-					<img src="${resource(dir: 'images/icons/silk', file: 'application_side_list.png')}" />&nbsp;
-					<warehouse:message code="default.list.label" args="['organization']"/>
-				</g:link>
-				<g:link class="button" action="create">
-					<img src="${resource(dir: 'images/icons/silk', file: 'add.png')}" />&nbsp;
-					<warehouse:message code="default.add.label" args="['organization']"/>
-				</g:link>
-			</div>
+            <div class="button-bar">
+                <g:link class="button" action="list">
+                    <img src="${resource(dir: 'images/icons/silk', file: 'application_side_list.png')}" />&nbsp;
+                    <warehouse:message code="default.list.label" args="['organizations']"/>
```

Comment:
Just as above, why don't we use args="[g.message(code:'organizations.label')]" instead?